docs/_pages/subjectguide/biology.md

```diff
@@ -0,0 +1,136 @@
+---
+permalink: /subject-guide/biology/
+title: RSE Toolkit - Subject Guide
+breadcrumb: Subject Guide
+layout: single
+---
+
+# Biology
+
+## Bioinformatics
+
+Major languages:
+
+for tools and also web-development efforts
+* Python
+* R
+* Java
+* minor c++
+* lately some poeple are big advocats of Rust
+
+for web development,a lot of different web technology stacks are used ranging for older stacks to the latest and greatest with React.js, Vue, etc
+Q: should we try to get some ideas on what is used most? I'll check if I can find some overview publication (eWorkbench?)
+
+A lot of emphasis is put on enabling Reproducible Research lately. This includes container technologies like Docker and workflow systems like Nextflow, snakemake which are the most popular ones.
+
+### Typical journals
+
+A lot of work is published in the following journals:
+
+* Bioinformatics
+* NAR
+* PLoS
+* eLife
+* JOSS
+
+Furthermore, an enormeous amount of articles about bioinformatics software are published on bioarxiv before actual publication via the said journals.
+
+
+### Resources
+
+* https://github.com/danielecook/Awesome-Bioinformatics
+* https://www.reddit.com/r/bioinformatics/
+* https://global.oup.com/academic/product/introduction-to-protein-science-9780198716846
+* https://web.stanford.edu/~hastie/ElemStatLearn/
+* https://www.biostars.org/
+* an overview of curated tools from the literature https://bio.tools/
+
+#### Python
+- pypi
+- (bio)conda
+- https://biopython.org/
+
+#### R
+- CRAN/MRAN
+- Bioconductor
+
+#### Java
+- mvn
+- https://biojava.org/
+
+#### C++
+- conan.io
+- http://biocpp.sourceforge.net/
+
+#### Rust
+
+
+### Introductory books
+- https://global.oup.com/academic/product/introduction-to-bioinformatics-9780198794141
+
+### Communities
+- ELIXIR
+
+### commercial companies
+- Qiagen
+- 10X Genomics
+- https://en.wikipedia.org/wiki/List_of_bioinformatics_companies
+
+
+## Bioimaging
+
+Major languages:
+
+for tools and also web-development efforts
```

Comment:
~~for tools and also web-development efforts~~ For tools and also web development efforts: